types/ink-box/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "jsx": "react",
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
Please remove this for backward compatibility, see comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44595#discussion_r425400276
Note: this would require rewrite of your imports to:
```tsx
import * as Box from "ink-box";
```
(this is only for DT tests, users can still use `esModuleInterop` at will)